vignettes/kpitools.Rmd

```diff
@@ -245,15 +245,16 @@ risklabs(c(0, 20, 30, 40, 50))
 risklabs(c(0, 20, 30, 40, 50, 60))
 ```
 
-It's simple to override these labels with the `risklabels` argument (be careful combining `risklabels` and `direction` arguments as the label order will be inverted). 
+It's simple to override these labels with the `risklabels` argument (be careful combining `risklabels`, `riskcolors` and `direction` arguments as the label order will be inverted). 
```

Comment:
better would be: "be careful combining `risklabels` and/or `riskcolors` with the `direction` arguments"